components/bl-list-component/styles/index.less

```diff
@@ -0,0 +1,25 @@
+@bl-customComponent-list-theme: @themePrimary;
+@bl-customComponent-list-themeTextColor: @appTextColor;
+
+@bl-customComponent-list-item-hover-background-color: if(@isLightTheme, rgba(0, 0, 0, 0.04), rgba(255, 255, 255, 0.04));
+
+@bl-customComponent-list-item-size: 20px;
```

Comment:
list-item-size -> list-item-fontSize

Reply:
this is a general parameter for size control, below is used to set the margins/paddings. Perhaps, in this case, it would be better to rename to indent or leave it as it is.

Reply:
So we must create variables for each of this properties